{"name": "Проверка принятия заказа курьером с неверным номером курьера", "status": "passed", "description": "Код операции:404, ответ:\"Курьера с таким id не существует\"", "start": 1729850893028, "stop": 1729850893787, "uuid": "59253d17-6041-4825-9a69-a6c8ae5f99c2", "historyId": "323a772e6d4e3f862e1c726da8bb5e0b", "testCaseId": "323a772e6d4e3f862e1c726da8bb5e0b", "fullName": "tests.order.test_take_order.TestTakeOrders#test_takin_order_wrong_curier_id_negative", "labels": [{"name": "parentSuite", "value": "tests.order"}, {"name": "suite", "value": "test_take_order"}, {"name": "subSuite", "value": "TestTakeOrders"}, {"name": "host", "value": "DESKTOP-03BV0KK"}, {"name": "thread", "value": "7868-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.order.test_take_order"}]}